CONTRIBUTING.md

```diff
@@ -1,28 +1,71 @@
 # Contributing
 
-## Contributor License Agreement (CLA)
-
-To start contributing to SumUp Open Source projects, please accept our [Contributor License Agreement](https://opensource.sumup.com/cla). Should you have any questions or concerns, please get in touch with [opensource@sumup.com](mailto:opensource@sumup.com).
-
 ## Code of Conduct (CoC)
 
 We want to foster an inclusive and friendly community around our Open Source efforts. Like all SumUp Open Source projects, this project follows the Contributor Covenant Code of Conduct. Please, [read it and follow it](https://github.com/sumup-oss/circuit-ui/tree/main/CODE_OF_CONDUCT.md).
 
 If you feel another member of the community violated our CoC or you are experiencing problems participating in our community because of another individual's behavior, please get in touch with our [maintainers](README.md#maintainers). We will enforce the CoC.
 
-## Workflows
+![Overview diagram of the contribution model](https://github.com/sumup-oss/circuit-ui/tree/main/assets/contribution-model.jpg?raw=true)
+
+## Submitting an issue
+
+In most cases, submitting an issue is the first step to contributing to Circuit UI. Check the existing issues and verify that your issue is not already submitted. Then use one of the templates to [ask a question](https://github.com/sumup-oss/circuit-ui/issues/new?template=question.md), [report a bug](https://github.com/sumup-oss/circuit-ui/issues/new?template=report-a-bug.md), [amend an existing component](https://github.com/sumup-oss/circuit-ui/issues/new?template=amend-existing-component.md), or [propose a new one](https://github.com/sumup-oss/circuit-ui/issues/new?template=propose-new-component.md).
+
+SumUp employees can also reach out in our internal Slack channel #design-system-web, although we strive to keep all communication in the open.
+
+## Submitting a pull request
+
+We appreciate pull requests (PRs) for smaller changes and bug fixes. For larger changes, we encourage you to [submit an issue](https://github.com/sumup-oss/circuit-ui/issues/new) to collect feedback first. The normal workflow looks as follows:
+
+1. Let others know that you're working on an issue by leaving a comment or assigning it to yourself.
+2. Clone the repository (you might need to fork it first) and branch out from the latest `main` branch.
```

Comment:
Shouldn't it still be `canary`? Or are we moving everything to `main`?

Reply:
We're moving everything to main. `canary` will still exist as a throw-away branch for pre-releases when we occasionally need them, but all feature branches should be based on and merged into `main`.

This is made possible by `changesets` which doesn't trigger a release on every commit, but only when instructed to.